package/android/src/main/java/com/mrousavy/camera/extensions/CameraCharacteristics+getPreviewSize.kt

```diff
@@ -0,0 +1,42 @@
+package com.mrousavy.camera.extensions
```

Comment:
The Preview size is also an output size, was there a specific reason you moved this to a separate file?

Reply:
Not at all, I'm new to _Kotlin_ and unfamiliar with the concept of Extensions file. If you think that both files should be merged back, I can 🤓